packages/stockflux-components/src/components/buttons/dialog-button/DialogButton.css

```diff
@@ -0,0 +1,43 @@
+.dialog {
+  display: inline-block;
+  color: var(--text-color);
+  border: none;
+  outline: none;
+  background-color: inherit;
+  cursor: pointer;
+}
+
+.dialog.small {
+  height: 30px;
+  width: 30px;
+  padding: 5px;
+  margin: 2px;
+}
+
+.dialog svg {
+  margin: auto;
+  display: block;
+  shape-rendering: auto;
```

Comment:
If the default value is `auto`, is this needed?

Reply:
that entire section of css was not needed as we have no svgs within the dialogbuttons, so i have removed it.